db/controller/articlesIdPatch.controller.js

```diff
@@ -0,0 +1,27 @@
+const { updateArticleById } = require("../models/articlesIdPatch.model");
+
+exports.patchArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+  if (typeof inc_votes !== "number") {
+    return res
+      .status(400)
+      .json({ error: "Bad request: 'inc_votes' must be a number" });
+  }
```

Comment:
Error handling shouldnt be done in the controller like this. We should do this in the model and throw the error so the .catch(...) below can pass it off using next to your middleware